server/views/grid/aligning/README.md

```diff
@@ -1 +1,3 @@
 Applying a class of either `grid--start`, `grid--center` or `grid--end` to the containing `grid` div will determine how grid cells are aligned horizontally.
+
+Applying a class of `grid--spaced` to the containing `grid` element will space out its child `grid__cell`s (useful for when e.g. there is an empty column of space between to content columns, rather than having to include an empty element for spacing).
```

Comment:
Yo.

I've also seen a couple of places in the C&T styleguide where the grid/gutter spacing has been deliberately ignored in favour of evenly-spread-columns-that-don't-add-up-to-12, so this would fit that use-case.